packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -865,7 +865,7 @@ export class Grapher
         // (2) needs to have data for the x and y dimension.
         let table = this.isScatter
             ? this.tableAfterAuthorTimelineAndActiveChartTransform
-            : this.inputTable
+            : this.tableAfterAuthorTimelineAndEntityFilter
```

Comment:
Since `tableAfterAuthorTimelineAndEntityFilter` is the 'cleaned' table that we actually want to use as a base table for Grapher, I think it makes sense to use it for the entity selector as well.